src/ng/directive/ngSwitch.js

```diff
@@ -134,10 +134,10 @@ var ngSwitchDirective = ['$animate', '$compile', function($animate, $compile) {
     require: 'ngSwitch',
 
     // asks for $scope to fool the BC controller module
-    controller: ['$scope', function ngSwitchController() {
+    controller: ['$scope', function NgSwitchController() {
      this.cases = {};
     }],
-    link: function(scope, element, attr, ngSwitchController) {
+    link: function(scope, element, attr, NgSwitchController) {
```

Comment:
Is this necessary (here and below). We usually name these parameters using camelCase.